docs/sources/explore/logs-integration.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Logs in Explore"
+description = "Logs in Explore"
+keywords = ["explore", "logs",]
+aliases = ["/docs/grafana/latest/features/explore/"]
+weight = 15
++++
+
+# Logs in Explore 
+
+Along with metrics, Explore allows you to investigate your logs in the following data sources:
+
+- [Loki](../datasources/loki)
+- [InfluxDB](../datasources/influxdb)
+- [Elasticsearch](../datasources/elasticsearch)
+
+During an infrastructure monitoring and incident response, you can dig deeper into the metrics and logs to find the cause. Explore also allows you to correlate metrics and logs by viewing them side-by-side. This creates a new debugging workflow where you can:
```

Comment:
```suggestion
During infrastructure monitoring or an incident response, you can dig deeper into metrics and logs to find the root cause. Explore also allows you to correlate metrics and logs by viewing them side-by-side. This creates a new debugging workflow where:
```

Reply:
I changed this last sentence so that it is stand-alone and complete before the :. This prevents a future translation bug.